plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -118,11 +117,26 @@ abstract QueryRunner createDeltaLakeQueryRunner(Map<String, String> connectorPro
 
     abstract List<String> listCheckpointFiles(String transactionLogDirectory);
 
+    protected void copyResources(String resourcePath, String target)
+    {
+        hiveMinioDataLake.getMinioClient().copyResourcePath(bucketName, resourcePath, target);
```

Comment:
`hiveMinioDataLake.getMinioClient().copyResourcePath` keeps appearing multiple times in the PR.

Would it make sense to expose in `HiveMinioDataLake` (same as it was done in `DockerizedMinioDataLake` the methods  :

- copyResources
- writeFile
- listFiles

?